src/components/ImportRecordsFlow/stepThree/assignFieldModal.js

```diff
@@ -72,13 +71,7 @@ export default function AssignFieldModal(props) {
             isOpen={isAssignFieldModalOpen}
             size="small"
             onRequestClose={onRequestClose}
-            footer={
-                <AssignFieldModalFooter
-                    onCancel={onRequestClose}
-                    onAssign={handleAssign}
-                    isAssignButtonDisabled={isAssignButtonDisabled}
```

Comment:
the intention here is to keep this button disabled only when the field is not assigned, then we need to fix the current behavior but also keep the behavior of having this button disabled when the field is not assigned